moto/ec2/models.py

```diff
@@ -3011,11 +3016,26 @@ def get_filter_value(self, filter_name):
         else:
             return super(VPC, self).get_filter_value(filter_name, "DescribeVpcs")
 
+    def modify_vpc_tenancy(self, tenancy):
```

Comment:
nitpick: Could be slightly rewritten (to make it more intuitive, checking the parameter assertions early in the method) to:
```
    def modify_vpc_tenancy(self, tenancy):
        if tenancy != "default":
            raise UnsupportedTenancy(tenancy)
        self.instance_tenancy = tenancy
        return True
```